src/modules/lua_mtev.c

```diff
@@ -129,6 +122,12 @@ typedef struct {
 } json_crutch;
 
 static void
+mtev_lua_push_timeval(lua_State *L, struct timeval time) {
```

Comment:
mtev.timeval.new is lua code that generates a timeval, right?
here I want to go the other way round: push a double to lua
I'm doing this because:
1) I saw other code doing this
2) I didn't see any lua code handling a timeval directly

In principle we could also implement a `setup_timeval` that pushes a userdata timeval with the corresponding __index function, but I think a double is fine.